src/models/organization.rs

```diff
@@ -26,6 +27,14 @@ pub struct OrganizationData<'c> {
     pub description: Option<String>,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(crate = "rocket::serde")]
+pub struct UpdateOrg {
+    pub name: String,
```

Comment:
Use reference instead of owned string to improve performance if this is provided by user.